system/src/system_task.cpp

```diff
@@ -310,7 +310,7 @@ void establish_cloud_connection()
         cellular_device_info(&device, NULL);
         if (device.dev == 8/*DEV_SARA_R410*/) {
             DEBUG("Device is SARA_R410, disabling Fast OTA!");
-            CLOUD_FN(spark_set_connection_property(particle::protocol::Connection::FAST_OTA, 0/*disabled*/, nullptr, nullptr), (void)0);
+            CLOUD_FN(spark_set_connection_property(particle::protocol::Connection::FAST_OTA, 1/*disabled*/, nullptr, nullptr), (void)0);
```

Comment:
Correct, I just flipped the 0 to 1 for quick Fast OTA testing.

We wouldn't need this code any longer, but I wonder if we should create a Wiring API to allow users to toggle this if they so desired.  Although after all of these fixes get implemented I can't think of a reason anyone would want to OTA update slower.  Once we test and it looks good though, we can just remove this code, since we have the history in github.  No need to even leave it commented out.